window-manipulate/window.js

```diff
@@ -0,0 +1,60 @@
+document.addEventListener("click", function(e) {
+
+  function callOnCurrentWindow(callback){
+    chrome.windows.getCurrent(
+      function(currentWindow) {
+        callback(currentWindow);
+      }
+    );
+  }
+
+  if (e.target.id === "window-update-size_768") {
+    callOnCurrentWindow((currentWindow) => {
+      updateInfo = {
```

Comment:
This and all other declarations should be scoped locally to prevent any unintended side effects. For example:
````
var updateInfo = {
}
````